pkg/runtime/nodes.go

```diff
@@ -38,6 +38,12 @@ const (
 	RemoteAddRoute                  = "seautil route add --host %s --gateway %s"
 	RouteOK                         = "ok"
 	LvscareStaticPodCmd             = `echo "%s" > %s`
+	CheckNetworkIsDefaultRoute      = "ip route show |grep default |grep %s"
+	DeleteRouteCmd                  = "if ip route |grep %s;then ip route del %s;fi"
+	AddRouteCmd                     = "ip route |grep %s || ip route add %s"
+	DeleteRouteFromFile             = "if [ -f /etc/sysconfig/network-scripts/route-%s ]; then sed -i \"/%s/d\" /etc/sysconfig/network-scripts/route-%s;fi"
+	AddStaticRouteFile              = "cat /etc/sysconfig/network-scripts/route-%s|grep %s || echo %s >> /etc/sysconfig/network-scripts/route-%s"
+	GetNetworkInterface             = "ip route |grep -v %s |grep %s| awk -F '[ \\t*]' '{print $3}'"
```

Comment:
Should use ip addr command to get it. grep node ip

Reply:
Some hosts do not have `ip` command, use "net.Interfaces()" to get network interface

Reply:
Some hosts do not have `ip` command. 
if remote: "ifconfig" cmd is system build-in. 
if local : use "net.Interfaces()".

Reply:
Or use sealutil to check interface and add router.